posthog-react-native/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# Next
+# Next (4.0.0 WIP)
```

Comment:
since `captureMode` is part of core, we need to release the next major for RN and Web

Reply:
https://github.com/PostHog/posthog-js-lite/pull/520